framework/src/headers/mac.rs

```diff
@@ -59,6 +62,19 @@ impl MacAddress {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct MacParseError(());
```

Comment:
Maybe we can just reuse this? https://doc.rust-lang.org/std/net/struct.AddrParseError.html